js/DropDownTree/Checkbox.md

```diff
@@ -1,15 +1,15 @@
 ---
 layout: post
-title: CheckBox with DropDownTree widget for Syncfusion Essential JS
-description: Describes about Checkbox functionalities in DropDownTree widget for Syncfusion Essential JS
+title: Checkbox with JavaScript DropDownTree Control | Syncfusion 
+description: Learn here about Checkbox functionalities in Syncfusion Essential JavaScript DropDownTree Control, its elements, and more.
 platform: js
 control: DropDownTree
 documentation: ug
 keywords: Checkbox, dropdown,textModes, Delimiter
 api: /api/js/ejDropDownTree
 ---
 
-# Checkbox
+# Checkbox with JavaScript DropDownTree widget
```

Comment:
Checkbox in. Use Checkbox in in any other place you used with

Reply:
Done